src/components/NcDateTimePicker/NcDateTimePicker.vue

```diff
@@ -259,6 +313,16 @@ const props = withDefaults(defineProps<{
 	 */
 	locale?: string
 
+	/**
+	 * The maximum date that can be selected.
+	 */
+	max?: Date
+
+	/**
+	 * The minimum date that can be selected.
+	 */
+	min?: Date
```

Comment:
Does it check the full date value, including time, or only the date?
If only the date, I'd rename prop to `minDate`.
In any case, I'd mention it in the prop docs.

Reply:
Full date value, the reason for calling it `max` and `min` is that they are what is used in the native date picker and in the HTML element for the same purpose